internal/dataplane/configfetcher/config_fetcher.go

```diff
@@ -34,9 +34,11 @@ type DefaultKongLastGoodConfigFetcher struct {
 	// - Services, Routes, and Consumers - based on their names. It ensures that IDs remain
 	// stable across restarts of the controller.
 	fillIDs bool
+	// workspace is the workspace name used in generating deterministic IDs. Only used when fillIDs = true.
+	workspace string
 }
 
-func NewDefaultKongLastGoodConfigFetcher(fillIDs bool) *DefaultKongLastGoodConfigFetcher {
+func NewDefaultKongLastGoodConfigFetcher(fillIDs bool, workspace string) *DefaultKongLastGoodConfigFetcher {
```

Comment:
`workspace` is passed but not stored as a struct field (as the linter discovered).

Reply:
fixed.